src/js/apps/admin/sidebar/action-sidebar_app.js

```diff
@@ -41,7 +41,12 @@ export default App.extend({
         this.stop();
       })
       .fail(({ responseJSON }) => {
-        Radio.request('alert', 'show:error', responseJSON.message);
+        const error = responseJSON.errors[0];
```

Comment:
Had an idea.  why don't we make a `Radio.request('alert', 'show:apiError', responseJSON)`. so that these are all simpler.  Then in that case I _do_ think we should loop over the errors array.  it wasn't necessary here, but it makes sense for other error types.